src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+@import './variables';
+@import './card';
+@import './stars';
+
+html {
+  margin: 0;
+  padding: 0;
+  font-family: $font-family-primary;
+  font-weight: $font-weight-regular;
+  font-style: normal;
+}
+
 body {
   margin: 0;
 }
```

Comment:
The `body` margin reset is redundant if it's already been defined in a CSS reset or normalize file. If you have such a file, ensure these styles are not duplicated.